helm/kamaji/templates/etcd_sts.yaml

```diff
@@ -40,11 +40,11 @@ spec:
             - --data-dir=/var/run/etcd
             - --name=$(POD_NAME)
             - --initial-cluster-state=new
-            - --initial-cluster=etcd-0=https://etcd-0.etcd.$(POD_NAMESPACE).svc.cluster.local:2380,etcd-1=https://etcd-1.etcd.$(POD_NAMESPACE).svc.cluster.local:2380,etcd-2=https://etcd-2.etcd.$(POD_NAMESPACE).svc.cluster.local:2380
+            - --initial-cluster={{ include "etcd.endpoints" . }}
```

Comment:
@prometherion please revert back to 

`--initial-cluster=etcd-0=https://etcd-0.etcd.$(POD_NAMESPACE).svc.cluster.local:2380,etcd-1=https://etcd-1.etcd.$(POD_NAMESPACE).svc.cluster.local:2380,etcd-2=https://etcd-2.etcd.$(POD_NAMESPACE).svc.cluster.local:2380`

The `--initial-cluster` option tells the initial cluster configuration for bootstrapping the cluster and it happens between `etcd` peers listening on port **2380** by default https://etcd.io/docs/v3.5/op-guide/configuration/#clustering.

The `etcd.overrides.endpoints` values tell how the `etcd` clients, e.g. the `tcp`, are connecting to the datastore and it happens on port **2379**